.github/workflows/pages-deployment.yaml

```diff
@@ -0,0 +1,33 @@
+on: [push]
```

Comment:
thought: If this happens on every commit, then I think that the existing `build.yml` is no longer required.

Reply:
Good point, I will remove as part of this PR.